`dotnet restore` incorrectly recommends installing aspire workload when nuget feeds are messed up

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Describe the bug

I was setting up a github action running an aspire test, and forgot to include the `nuget.config` to point to the nightly aspire feeds.  When I did this, `dotnet restore` tried to tell me I needed the aspire workload.

```
/usr/share/dotnet/sdk/9.0.200/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.ImportWorkloads.targets(38,5): error NETSDK1147: To build this project, the following workloads must be installed: aspire [/home/runner/work/aspire-pipeline/aspire-pipeline/AspireInGithubActions.AppHost/AspireInGithubActions.AppHost.csproj]
Error: /usr/share/dotnet/sdk/9.0.200/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.ImportWorkloads.targets(3[8](https://github.com/afscrome/aspire-pipeline/actions/runs/13765059067/job/38489473027#step:4:9),5): error NETSDK1147: To install these workloads, run the following command: dotnet workload restore [/home/runner/work/aspire-pipeline/aspire-pipeline/AspireInGithubActions.AppHost/AspireInGithubActions.AppHost.csproj]
Error: Process completed with exit code 1.
```

### Expected Behavior

`dotnet restore` should not suggest installing the now deprecated workload.

### Steps To Reproduce

https://github.com/afscrome/aspire-pipeline/actions/runs/13765059067

### Exceptions (if any)

https://github.com/afscrome/aspire-pipeline/actions/runs/13765059067/job/38489473027#step:4:38
Error: /usr/share/dotnet/sdk/9.0.200/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.ImportWorkloads.targets(38,5): error NETSDK1147: To build this project, the following workloads must be installed: aspire [/home/runner/work/aspire-pipeline/aspire-pipeline/AspireInGithubActions.AppHost/AspireInGithubActions.AppHost.csproj]
Error: /usr/share/dotnet/sdk/9.0.200/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.ImportWorkloads.targets(3[8](https://github.com/afscrome/aspire-pipeline/actions/runs/13765059067/job/38489473027#step:4:9),5): error NETSDK1147: To install these workloads, run the following command: dotnet workload restore [/home/runner/work/aspire-pipeline/aspire-pipeline/AspireInGithubActions.AppHost/AspireInGithubActions.AppHost.csproj]
Error: Process completed with exit code 1.

### .NET Version info

https://builds.dotnet.microsoft.com/dotnet/Sdk/9.0.200/dotnet-sdk-9.0.200-linux-x64.tar.gz

### Anything else?

_No response_

--- a/src/Aspire.AppHost.Sdk/SDK/Sdk.in.props
+++ b/src/Aspire.AppHost.Sdk/SDK/Sdk.in.props
@@ -4,5 +4,6 @@
     <!-- Do not import the workload targets as this project is instead using
     the Aspire.AppHost.Sdk -->
     <SkipAspireWorkloadManifest>true</SkipAspireWorkloadManifest>
+    <IsAspireHost>true</IsAspireHost>
   </PropertyGroup>
 </Project>

--- a/src/Aspire.ProjectTemplates/templates/aspire-apphost/9.2/Aspire.AppHost1.csproj
+++ b/src/Aspire.ProjectTemplates/templates/aspire-apphost/9.2/Aspire.AppHost1.csproj
@@ -7,7 +7,6 @@
     <TargetFramework>net8.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
-    <IsAspireHost>true</IsAspireHost>
     <UserSecretsId>98048c9c-bf28-46ba-a98e-63767ee5e3a8</UserSecretsId>
   </PropertyGroup>
 

--- a/src/Aspire.ProjectTemplates/templates/aspire-empty/9.2/AspireApplication.1.AppHost/AspireApplication.1.AppHost.csproj
+++ b/src/Aspire.ProjectTemplates/templates/aspire-empty/9.2/AspireApplication.1.AppHost/AspireApplication.1.AppHost.csproj
@@ -7,7 +7,6 @@
     <TargetFramework>net8.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
-    <IsAspireHost>true</IsAspireHost>
     <UserSecretsId>98048c9c-bf28-46ba-a98e-63767ee5e3a8</UserSecretsId>
   </PropertyGroup>
 

--- a/src/Aspire.ProjectTemplates/templates/aspire-starter/9.2/Aspire-StarterApplication.1.AppHost/Aspire-StarterApplication.1.AppHost.csproj
+++ b/src/Aspire.ProjectTemplates/templates/aspire-starter/9.2/Aspire-StarterApplication.1.AppHost/Aspire-StarterApplication.1.AppHost.csproj
@@ -7,7 +7,6 @@
     <TargetFramework>net8.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
-    <IsAspireHost>true</IsAspireHost>
     <UserSecretsId>98048c9c-bf28-46ba-a98e-63767ee5e3a8</UserSecretsId>
   </PropertyGroup>
 

